lib/practice_exercises.rb

```diff
@@ -1,13 +1,30 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  if list.length == 0
+    return 0
+  end
+  unique_items = 1
+  list.length.times do |index|
+    if index != 0 && list[index - 1] != list[index]
+      unique_items +=1
+    end
+  end
+  return unique_items
 end
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Did not have time to complete all the way
+
+# Time Complexity: O(n^2)
```

Comment:
Given what you have here I would say O(m * n) where m is the length of the 1st string and n is the number of strings.